.licenseignore

```diff
@@ -0,0 +1,16 @@
+Sources/CNIOAtomics/src/cpp_magic.h
```

Comment:
What's the format for this file? Presumably paths relative to the root of the repo, but are wildcards supported? Can whole directories be ignored (e.g. vendored in dependencies)?

Reply:
The format is almost `.gitignore` like. So glowing and everything is supported i.e. `**/Package.swift` works.